### PR TITLE
Dev_WizardInfoDialog

### PR DESCRIPTION
**Pull request by Philoul**
I replaced hard coded string of WizardInfoDialog (ISF, IC, U) with String format (same change than in WizardDialog : format_bg_isf, format_carb_ic and format_cob_ic)
I have to change a little bit "format_bg_isf" in string file...
=> I didn't change trend value (in WizardDialog this value is formated with units "mg/dl" or "mmol/l" but not in WizardInfoDialog...)

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1605,7 +1605,7 @@
     <string name="close">Close</string>
     <string name="increasingmaxbasal">Increasing max basal value because setting is lower than your max basal in profile</string>
     <string name="smscommunicator_messagebody">Invalid message body</string>
-    <string name="format_bg_isf">%1$s ISF: %2$.1f</string>
+    <string name="format_bg_isf">%1$.0f ISF: %2$.1f</string>
```
Comment (MilosKozak) [inline app/src/main/res/values/strings.xml]: this is already used as string

Comment (Philoul) [reply app/src/main/res/values/strings.xml]: Ok, it works with a string format but it's just not so pretty (one decimal for BG...), and in french I see a point separator (decimal separator is a comma...) 
i.e. "125.8 ISF: 50.0" becomes "125.8 SI: 50,0"

I checked this format works in WizardDialog but I did not think it was used as string elsewhere...

Currently, for BG there is a DecimalFormatter.to1Decimal (I think it's a mistake and DecimalFormatter.to0Decimal would be better). But I tried to add this formatter in second parameter of String.format function (bg value) but when I tried it AAPS froze (????).

I don't understand why line below doesn't work... (tested with original "string format" for 1rst param)
treatments_wizard_bg.text = String.format(MainApp.gs(R.string.format_bg_isf), DecimalFormatter.to0Decimal(JsonHelper.safeGetDouble(json, "bg")), JsonHelper.safeGetDouble(json, "isf"))

**Comment by Philoul** [comment]
It works with original String format... (line code not very "beautifull" but it works... (I put bg value with DecimalFormatter outside String.format and the "string" parameter is empty....

treatments_wizard_bg.text = DecimalFormatter.to0Decimal(JsonHelper.safeGetDouble(json, "bg")) + String.format(MainApp.gs(R.string.format_bg_isf), "", JsonHelper.safeGetDouble(json, "isf"))

**Comment by MilosKozak** [comment]
i was trying to use .0 for mgdl and .1 for mmol in displaying BG

**Comment by MilosKozak** [comment]
PR is messed up noe.
Do you still have sources?

**Comment by Philoul** [comment]
Yes, I have deleted my fork (too dirty!). I close this PR and create a new one (I have made a backup of my changes...)

**Pull request closed without merging**